app/views/users/index.html.erb

```diff
@@ -1 +1,16 @@
-<h1>Username</h1>
\ No newline at end of file
+<ul>
+  <% @users.each do |user| %>
+    <li class="card">
+      <div class="photo">
+        <%= image_tag(user.photo, width: "150px") %>
+      </div>
+      <div class="user_card">
+        <div class="user_name"><%= user.name %></div>
+        <div class="user_posts">Number of posts: <%= user.posts_counter %></div>
+      </div>
+      <button id="user_btn">
+        <%= link_to "Access User Info", user_path(user) %>
```

Comment:
@whiz25 the link you highlighted taking you to http://127.0.0.1:3000/users/users_id which has info about a user and their posts which is working fine if you click the link.
If you mean all posts for one user, there is a button (See all posts) which if you click it shows all posts for a user.
Please update me with some more detail because I have no idea what you intended to say.